<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>yellow-tomato</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/tomato.qrc">
    <normaloff>:/favicon.jpg</normaloff>:/favicon.jpg</iconset>
  </property>
  <widget class="AnalogClock" name="analogClock" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>The current time</string>
   </property>
   <property name="whatsThis">
    <string>The analog clock widget displays the current time.</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress30">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress1440">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="maximum">
    <number>1440</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDone">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>280</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelStockName">
      <property name="text">
       <string>上证指数最新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdStockNow">
      <property name="digitCount">
       <number>7</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelStockName_2">
      <property name="text">
       <string>昨收</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcdStockLast">
      <property name="digitCount">
       <number>7</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalogClock</class>
   <extends>QWidget</extends>
   <header>analogclock.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource/tomato.qrc"/>
 </resources>
 <connections/>
</ui>
